toolkit/scripts/toolchain/download_toolchain_rpm.sh

```diff
@@ -104,6 +116,13 @@ if [ -n "$key" ]; then
     key="--private-key=$key"
 fi
 
+if $enforce_signatures; then
+    if [ -z "$allowable_gpg_keys" ]; then
```

Comment:
I'd even test that each of the key paths is an existing file with `[ -f ... ]` to catch issues earlier and print a more helpful error message.